spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/condition/SpringBootCondition.java

```diff
@@ -37,7 +37,7 @@
  */
 public abstract class SpringBootCondition implements Condition {
 
-	private final Log logger = LogFactory.getLog(getClass());
+	protected final Log logger = LogFactory.getLog(getClass());
```

Comment:
@snicoll FWIW, I thought of that and did check all references I could find, but couldn't find any at least in spring-boot space.
As for the `getLogger()`, I am ok with that, will update pr